### PR TITLE
Visual & Front-end Refresh

### PR DESCRIPTION
**Pull request by sophshep**
This PR adds in some of the [reusable styles](https://github.com/styleguide/css/styles) that @github/design-systems has created so that we can bring this site in line with other GitHub marketing styles. 

This effort improves the site in many places:

- Use [npm to bring in Primer modules](https://github.com/styleguide/css/getting-started), which means we'll be able to easily update the styles as they evolve in the future
- Deletes [a lot of custom CSS](https://github.com/github/government.github.com/pull/524/files#diff-72abf624385ed0401c0c530691c3ac3b) & reduces the final, compiled css file from 137kb to 88kb 🎉 
- Deletes all legacy Bootstrap code that was being used
- Updates typography to use Roboto, sizing scale, and colors to more closely match GitHub.com/home
- Use Primer grid system, which improves layouts on small & midsize screens 
- Update octicons to use the Jekyll helper & SVGs instead of the deprecated icon font
- Replace logo with SVG so it's crisp on all screens

| Page URL  | Before  | After  |
|---|---|---|
| https://government.github.com/ | image | image |
| https://government.github.com/community/ | image | image |
| https://government.github.com/accessibility/ | image | image |
| https://government.github.com/#contact-a-human | image | image |

### CODE REVIEW

**Comment by jbjonesjr** [comment]
:+1:

**Comment by jbjonesjr** [comment]
👋 @sophshep , i'm going to merge some other PRs, I'll try and remember to come back and update your branch later :)

**Comment by broccolini** [comment]
This looks like a fun pr! Let me know when you're ready for review or if there's anything I can help with. Curious if you learn anything from this that can feed back into our installation docs 💭 .

**Comment by sophshep** [comment]
> Let me know when you're ready for review or if there's anything I can help with. Curious if you learn anything from this that can feed back into our installation docs 💭 .

@broccolini It's ready for a review from Design Systems. You'll be happy to hear that the installation went smoothly! I followed @jonrohan's instructions and didn't curse once. 😄

File: _includes/footer.html [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
-      <div class="site-footer clearfix">
-        <ul class="site-footer-links right no-style-list">
-          <li class="no-mobile"><a href="https://help.github.com/articles/github-glossary" target="_blank">Glossary</a></li>
-          <li class="no-mobile"><a href="https://help.github.com/security">Security</a></li>
-          <li><a href="mailto:government@github.com">Contact Us</a></li>
+    <footer class="border-top">
+      <div class="container-lg px-3 py-4 clearfix">
+        <ul class="list-style-none float-right text-small">
+          <li class="d-none d-lg-inline-block ml-lg-2"><a href="https://help.github.com/articles/github-glossary" target="_blank">Glossary</a></li>
+          <li class="d-none d-lg-inline-block ml-lg-2"><a href="https://help.github.com/security">Security</a></li>
```
Comment (nschonni) [inline _includes/footer.html]: https://help.github.com/articles/github-security/ from #453

File: _includes/footer.html [hunk 2]
```diff
@@ -1,23 +1,25 @@
-    <footer class="container">
-      <div class="site-footer clearfix">
-        <ul class="site-footer-links right no-style-list">
-          <li class="no-mobile"><a href="https://help.github.com/articles/github-glossary" target="_blank">Glossary</a></li>
-          <li class="no-mobile"><a href="https://help.github.com/security">Security</a></li>
-          <li><a href="mailto:government@github.com">Contact Us</a></li>
+    <footer class="border-top">
+      <div class="container-lg px-3 py-4 clearfix">
+        <ul class="list-style-none float-right text-small">
+          <li class="d-none d-lg-inline-block ml-lg-2"><a href="https://help.github.com/articles/github-glossary" target="_blank">Glossary</a></li>
```
Comment (nschonni) [inline _includes/footer.html]: Trailing `/`

File: _includes/header.html [hunk 3]
```diff
@@ -6,23 +6,23 @@
...
-              <li><a href="/accessibility">Accessibility</a></li>
-              <li><a href="/#contact-a-human">Contact</a></li>
-            </ul>
-        </div>
+  <div class="page-header pt-4 pb-5">
+    <div class="container-lg">
+      <div class="text-center">
+        <a href="/"><img class="logo py-3" src="/assets/img/logomark.svg" alt="GitHub and Government"></a>
+        <ul>
+          <li class="text-center d-block d-sm-inline-block px-1 px-md-3"><a href="/community/" class="text-white">Who's using GitHub</a></li>
```
Comment (pmarsceill) [inline _includes/header.html:21]: Consider using a smart apostrophe.

**Comment by mdo** [comment]
Drive by feedback—the talk to a human section feels a bit lackluster and the copy 2x2 group of content above it feels off on the spacing.

**Comment by jbjonesjr** [comment]
cc/ @m0rgane

**Comment by jbjonesjr** [comment]
Does some of @benbalter's work in https://github.com/government/best-practices/pull/145 also prove relevant here?

**Review by jbjonesjr — APPROVED** [review]

**Comment by nschonni** [comment]
You can just cherrypick or redo #589 if you want to fix the build

**Comment by sophshep** [comment]
I've updated everything in this PR to the latest version of Primer, which makes the colors accessible 🎉 

It doesn't include many design changes, but I think this is good to merge and we can address anything else as followups.

**Review by benbalter — APPROVED** [review]

**Pull request merged**